Connectors/Tekla/Speckle.Connector.TeklaShared/SpeckleTeklaPanelHost.cs

```diff
@@ -24,10 +24,12 @@ public SpeckleTeklaPanelHost()
     this.Text = "Speckle (Beta)";
     this.Name = "Speckle (Beta)";
 
+    string version = GetVersion().ToString()[1..]; // removes the 'v' from version
```

Comment:
seems odd. Why we need version to be able get icon, isn't it already living in shared project?

Reply:
If we need the icon here too, we might as well just embed it as an EmbeddedResource accessible through `Properties.Resources.My_BitMap`